packages/core/src/operator/operator-runtime/validateConfirmData.ts

```diff
@@ -0,0 +1,144 @@
+import { operatorState } from "./operatorState.js";
+import { Challenge, config, getConfirmDataAndHash, PublicNodeBucketInformation, verifyChallengerSignedHash } from "../../index.js";
+import axios from "axios";
+import { ethers } from "ethers";
+
+
+export async function validateConfirmData(currentChallenge: Challenge, subgraphIsHealthy: boolean, event?: ethers.EventLog): Promise<boolean> {
+
+    try {
```

Comment:
We should not have these try catch arount the whole function rather catch each possible error by itself like done bellow.

It seems, the only error this try would catch is in `await getConfirmDataAndHash(assertionIds, subgraphIsHealthy)`.